lists/bd/bd-nbr.json

```
@@ -0,0 +1,53 @@
+{
+  "name": {
+    "en": "National Board of Revenue",
+    "local": "জাতীয় রাজস্ব বোর্ড"
+  },
+  "url": "https://nbr.gov.bd/",
+  "description": {
+    "en": "The National Board of Revenue is the authority for tax administration in Bangladesh, including customs excise, VAT, and income tax. Organisations and companies registered with the NBR will be issued with a Value Added Tax Registration Certificate containing their BIN registration number. "
+  },
+  "coverage": [
+    "BD"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "charity",
+    "trust"
+  ],
+  "sector": [],
+  "code": "BD-NBR",
+  "confirmed": false,
```

```suggestion
  "confirmed": true,
```

We set this to true at the point of submitting the PR... and treat the PR process as the final rubber stamp.